main.go

```diff
@@ -4,54 +4,16 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/fullstorydev/hauser/client"
```

Comment:
These look wrong. Try `$FS_HOME/bin/goimports -local github.com/fullstorydev/hauser/ -w .`

Reply:
I fixed the grouping if that's what looks wrong.  Given that you used the -local option, I'm assuming that was the problem.